functions-runtime/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM denoland/deno:1.25.3
+
+COPY entrypoint.sh /entrypoint.sh
+
+USER deno
+
+ENTRYPOINT /entrypoint.sh
```

Maybe we want to use the following so it is not run in a separate shell and SIGTERM etc are correctly passed to the deno runtime
```suggestion
ENTRYPOINT ["deno", "run", "--allow-net", "$SCRIPT"]
```

---

As we could add some things at startup time via the entry point script, I'll keep it this way now.